lighthouse-cli/test/fixtures/seo/seo-tester.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<!--
+ * Copyright 2017 Google Inc. All rights reserved.
```

Comment:
small license?

Reply:
> small license?

our HTML files don't appear to have a version of the small license and I didn't care enough to make one. Good first bug! :)

Reply:
there u go: https://github.com/GoogleChrome/lighthouse/pull/3309/files

Reply:
😶